bridge/test/SideNFTToken_test.js

```diff
@@ -0,0 +1,34 @@
+const SideToken = artifacts.require('./SideNFTToken');
+
+const utils = require('./utils');
+
+contract('SideNFTToken', async function (accounts) {
+    const tokenCreator = accounts[0];
+    const tokenName = "The Drops";
+    const tokenSymbol = "drop";
+    const tokenBaseURI = "ipfs:/";
+
+    before(async function () {
+        await utils.saveState();
+    });
+
+    after(async function () {
+        await utils.revertState();
+    });
+
+    describe('constructor', async function () {
+
+        it('should create side token', async function () {
+            let token = await SideToken.new(tokenName, tokenSymbol, tokenCreator, tokenBaseURI);
+            assert.isNotEmpty(token.address)
+        });
+        it('should fail empty minter address', async function () {
+            await utils.expectThrow(SideToken.new(tokenName, tokenSymbol, '0x', tokenBaseURI));
+        });
+
+    });
+
+    // TODO: add tests associated to permit (SideToken_test.js for reference).
+
```

Comment:
Also this tests should include add minting Tests and transfer test to a wallet, a contract that does not implement ERC165 interfaces, and one that does
We could copy paste some of the open zeppelin tests for this. If we consider it an over kill, we can skip this until we are working on the MVP and have the ok from the security team to skip this tests

Reply:
Just talked with Marccio, we will leave this tests when we start using the transfer and minting when we tackle the claim task